3-UsingAPIs/Week2/homework/ex2-pokemonApp/index.js

```diff
@@ -1,39 +1,72 @@
 'use strict';
-/*------------------------------------------------------------------------------
-Full description at: https://github.com/HackYourFuture/Homework/blob/main/3-UsingAPIs/Week2/README.md#exercise-2-gotta-catch-em-all
+async function fetchData(url) {
+  const response = await fetch(url);
+  if (!response.ok) {
+    throw new Error(`${response.status}:${response.statusText}`);
+  }
+  return response.json();
+}
 
-Complete the four functions provided in the starter `index.js` file:
+function fetchAndPopulatePokemons(data) {
+  data.results.forEach((element) => {
+    const option = document.createElement('option');
+    option.value = element.name;
+    option.textContent = element.name;
+    document.querySelector('select').appendChild(option);
+  });
```

Comment:
Nice! You can also choose to "destructure" the `name` from `element` because that's the only property you're using. You don't have to change this, just a tip 🙂 

```suggestion
  data.results.forEach(({ name ) => {
    const option = document.createElement('option');
    option.value = name;
    option.textContent = name;
    document.querySelector('select').appendChild(option);
  });
```